strr-strata-web/app/stores/strataApplication.ts

```diff
@@ -65,6 +65,10 @@ export const useStrrStrataApplicationStore = defineStore('strr/strataApplication
   const submitStrataApplication = async () => {
     const body = createApplicationBody()
 
+    // TODO: confirm if jobTitle for completing party is required for strata application
+    // @ts-expect-error - temporary until we confirm requirements
```

Comment:
yes I already reached out to lekshmi and she said she's going to revert this change so that its not required for completing party